app/deck/1.7.x/guides/environment-variables.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Using environment variables with decK
+toc: false
+---
+
+When you use decK to apply configurations to {{site.base_gateway}},
+decK reads data in plain text from a state file by default. To improve security, you 
+can also store sensitive information, for example `apiKey` or `client_secret`, in 
+environment variables. decK can then read data directly from the environment
+variables and apply it.
+
+Create environment variables with the `DECK_` prefix and reference them as 
+`{%raw%}${{ env "DECK_*" }}{%endraw%}` in your state file.
+
+The following example demonstrates how to apply an apiKey stored in an environment variable. 
+You can use this method for any sensitive content. 
+
+1. Create an environment variable:
+    <div class="copy-code-snippet"><pre><code>export DECK_API_KEY=<div contenteditable="true">{API_KEY}</div></code></pre></div>
+
+2. Save the following snippet into a `env-demo.yaml` file:
+
+    ```yaml
+    _format_version: "1.1"
+    consumers:
+    - keyauth_credentials:
+      - key: ${{ env "DECK_API_KEY" }}
```

Comment:
```suggestion
      - key: {%raw%}${{ env "DECK_API_KEY" }}{%endraw%}
```
Didn't catch this the first time, but it's currently not displaying in the output.